lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -698,7 +711,7 @@ def update_task_progress():
     report_store = ReportStore.from_config()
```

Comment:
sure, I was only refactoring it so I can include the test (which fails in the original version)--but I don't really need to